types/office-js-preview/index.d.ts

```diff
@@ -17209,7 +17211,9 @@ declare namespace Office {
          * **Important**:
          *
          * - Only one notification of type `InsightMessage` is allowed per add-in.
-         * (The `InsightMessage` type is currently in preview.) Attempting to add more will throw an error.
+         * (The `InsightMessage` type was introduced in 
+         * {@link https://docs.microsoft.com/javascript/api/requirement-sets/outlook/requirement-set-1.10/outlook-requirement-set-1.10 | Mailbox requirement set 1.10}.)
```

Comment:
I'm not sure you need this note about InsightMessage being part of 1.10. Does that type have its own documentation?

Reply:
Thanks for the feedback. I wanted to distinguish introduction of support between `InsightMessage` and other notification types (support was introduced in 1.3). I do see your point though, especially since `InsightMessage` has it [own documentation](https://docs.microsoft.com/javascript/api/outlook/office.mailboxenums.itemnotificationmessagetype). I could link it to that instead and remove mention of 1.10 here.